package.json

```diff
@@ -15,7 +15,8 @@
   "scripts": {
     "setup": "yarn install && yarn allow-scripts",
     "build": "run-p --print-label build:freighter-api build:docs build:extension",
-    "build:netlify": "cd docs && yarn --legacy-peer-deps && yarn build",
+    "build:netlify": "cd docs && npm pkg set 'dependencies.@stellar/freighter-api'='latest' && yarn --legacy-peer-deps && yarn build",
```

Comment:
setting freighter-api to "latest" will force yarn to get what ever has been published to npm